Sources/NIOSSL/TLSConfiguration.swift

```diff
@@ -84,6 +84,55 @@ public enum NIORenegotiationSupport {
     case always
 }
 
+/// Signature algorithms. The values are defined as in TLS 1.3
+public enum SignatureAlgorithm : UInt16 {
+    case rsaPkcs1Sha1 = 0x0201
+    case rsaPkcs1Sha256 = 0x0401
+    case rsaPkcs1Sha384 = 0x0501
+    case rsaPkcs1Sha512 = 0x0601
+    case ecdsaSha1 = 0x0203
+    case ecdsaSecp256R1Sha256 = 0x0403
+    case ecdsaSecp384R1Sha384 = 0x0503
+    case ecdsaSecp521R1Sha512 = 0x0603
+    case rsaPssRsaeSha256 = 0x0804
+    case rsaPssRsaeSha384 = 0x0805
+    case rsaPssRsaeSha512 = 0x0806
+    case ed25519 = 0x0807
+}
+
+
+/// The default list of accepted signature algorithms for verifying, taken from t1_lib.cc
+public let defaultVerifySignatureAlgorithms : [SignatureAlgorithm] = [
+    // List our preferred algorithms first.
+    .ecdsaSecp256R1Sha256, .rsaPssRsaeSha256, .rsaPkcs1Sha256,
+    
+    // Larger hashes are acceptable.
+    .ecdsaSecp384R1Sha384, .rsaPssRsaeSha384, .rsaPkcs1Sha384,
+    
+    .rsaPssRsaeSha512, .rsaPkcs1Sha512,
+    
+    // For now, SHA-1 is still accepted but least preferable.
+    .rsaPkcs1Sha1
+]
```

Comment:
I would really love to find a way to avoid hard-coding this, as swift-nio-ssl doesn’t have its own preferences about what to use here. Rather than use an explicit default of this kind, let’s default to `nil` instead.